### DISCUSSION TITLE
Spot Instance selection

### DISCUSSION THREAD

**Discussion by theophileds** [runs-on/runs-on#77] [Q&A] [answered]
When specifying multiple instance families using the family setting in GitHub Actions, how are spot instances selected? Does it choose the cheapest available option? 

Additionally, is there a difference between using `family=c7a+c7i+m7a` versus `family=c7a,c7i,m7a`?

**Comment by crohr** [comment, marked as answer]
The fleet is created with `capacity-optimized` - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-fleet-allocation-strategy.html

At this time `capacity-optimized-prioritized` is not supported for instant fleets.

So currently the order should not necessarily mean anything. It just expands the number of potential pools from which an instance is selected.

**Reply by theophileds** [reply]
Thanks for the clarification!

**Comment by crohr** [comment]
Spot allocation strategy will be able to be customised starting with v2.3.0 - https://github.com/runs-on/runs-on/pull/87
